res/controllers/Denon-MC7000-scripts.js

```diff
@@ -37,7 +37,7 @@ var MC7000 = {};
 // 1) Beat LED in Slicer mode (counts every 8 beats AFTER the CUE point)
 //    only works for CONSTANT TEMPO tracks
 //    needs beat grid and CUE point set
-MC7000.experimental = false;
+MC7000.experimental = true;
```

Comment:
I assume this was added on accident?

Reply:
Oh yes. I have got it enabled all time but for public need to disable. Gonna remember next time... Thanks!